userspace/libscap/scap.h

```diff
@@ -171,6 +171,7 @@ typedef enum scap_fd_type
 	SCAP_FD_BPF = 17,
 	SCAP_FD_USERFAULTFD = 18,
 	SCAP_FD_IOURING = 19,
+  SCAP_FD_MEMFD = 20
```

Comment:
indentation

Reply:
Well, this is weird!! why isn't my spaces being formatted correctly? Looks fine in my editor!!
<img width="233" alt="Screenshot 2023-06-25 at 11 43 46 AM" src="https://github.com/falcosecurity/libs/assets/55309324/dab99227-add5-4985-98bd-fb58488dd4af">